pkg/toml/util.go

```diff
@@ -0,0 +1,48 @@
+package toml
+
+import (
+	"os"
+	"fmt"
+	"github.com/naoina/toml"
+	"github.com/SeerUK/tid/pkg/types"
+)
+
+// BoltDatabaseFilename is the name of the database file name on disk.
+const TomlConfigFilename = "config.toml"
+
+// Open opens the configuration file or it creates it if it doesn't exist already.
+func Open(tidDir string) (types.TomlConfig, error) {
+	config := types.NewTomlConfig()
+
+	// Make the `path` if it does not exist.
+	err := os.MkdirAll(tidDir, os.ModePerm)
+	if err != nil {
+		return config, err
+	}
+
+	var configFilePath = fmt.Sprintf("%s/%s", tidDir, TomlConfigFilename)
```

Comment:
this may not be compatible on windows! The standard library offers a more robust method for joining file paths.